client/device/asa/update.go

```diff
@@ -115,7 +135,17 @@ func Update(ctx context.Context, client http.Client, updateInp UpdateInput) (*Up
 		return nil, err
 	}
 
-	if err := retry.Do(ctx, UntilStateDoneAndConnectivityOk(ctx, client, outp.Uid), retry.DefaultOpts); err != nil {
+	if err := retry.Do(
+		ctx,
+		UntilStateDoneAndConnectivityOk(ctx, client, outp.Uid),
+		retry.NewOptionsBuilder().
+			Message("waiting for ASA to be done with connectivity status OK").
```

Comment:
Waiting for ASA to reach connectivity state ONLINE - please change in other locations for other device types too

Reply:
done 👍